### ISSUE TITLE
Discrepancy between displayed text and aria-label for local_time_ago

### ISSUE THREAD

**Issue by dchacke** [basecamp/local_time#114]
Using `local_time_ago`, the displayed text updates automatically as time passes, but the corresponding `aria-label` remains static, causing a discrepancy between the two.

<img width="709" alt="display text vs aria" src="https://user-images.githubusercontent.com/2866749/154183107-3c7a7212-1c5b-4247-9262-e65aeff765ac.png">

Expected behavior: the `aria-label` should be updated alongside the displayed text.

EDIT: Here's the HTML from the screenshot above (line breaks added for readability):

```html
<time datetime="2022-02-16T01:57:56Z"
      data-local="time-ago"
      title="February 15, 2022 at 5:57pm PST"
      aria-label="7 minutes ago">
  11 minutes ago
</time>
```

**Comment by josefarias** [comment]
Thanks for reporting. We plan to remove `aria-label`s since `<time>` elements don't need them, so this will go away soon.
